File: Curso_de_Programacion_Orientada_a_Objetos-POO/PHP/uberX.php
<?php
require_once("car.php");

class UberX extends Car {
  public $brand;
  public $model;

  public function __construct($license, $driver, $brand, $model){
    parent::__construct($license, $driver);
    $this->brand = $brand;
    $this->model = $model;
  }

  public function printDataUberX(){
    echo "Licencia: $this->license, Driver: " , $this->driver->name , ", Marca: " , $this->brand , ", Modelo: " . $this->model;
  }
}
?>

File: fundamentos_POO-Programacion_orientada_objetos-Multilenguaje/Java/UberX.java
class UberX extends Car {
  String brand;
  String model;

  public UberX(String license, Account driver, String brand, String model) {
    super(license, driver);
    this.brand = brand;
    this.model = model;

  }

  @Override
  void printDataCar() {
    System.out.println(
        "Car licence: " + getLicense() + " Driver: " + getDriver().name + "Documento: " + getDriver().document
            + " Pasajeros: " + getPassenger() + " Marca: " + brand + " Modelo: " +
            model);
  }
}

// void printDataUberX() {
// System.out.println(
// "Car licence: " + getLicense() + " Driver: " + getDriver().name + "Documento:
// " + getDriver().document
// + " Pasajeros: " + getPassenger() + " Marca: " + this.brand + " Modelo: " +
// this.model);
// }
